Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM omegacen/conda-builder
+
+LABEL maintainer="H.Buddelmeijer@rug.nl"
+LABEL version="1.0"
+LABEL description="Image for building LaTeX documentation through tectonic"
+
+ARG CONDA_PREFIX=/opt/conda
+
+COPY example.tex /root/example.tex
+
+RUN ${CONDA_PREFIX}/bin/conda install -c conda-forge tectonic texlive-core \
+    && ${CONDA_PREFIX}/bin/tectonic /root/example.tex
```

Comment:
I'm not sure this is a common idiom in Dockerfiles -- including a test stage I mean. Should this not be tested somewhere else, e.g. on the build Docker image?

Reply:
I've added a comment for clarification. The goal of compiling the example is to fetch LaTeX packages. The image is used in the CI of https://gitlab.astro-wise.org/micado/calibrationplan, and pre-fetching these packages reduces the CI time from 5 minutes to 1. There doesn't seem to be another way to fetch the packages.